### ISSUE TITLE
Issues mentioned in DupeFix Project

### ISSUE THREAD

**Issue by IcarussOne** [TheCodex6824/ThaumcraftFix#9]
As all the duplication fixes from Universal Tweaks are already in which were from DupeFix originally, there are at least two mentioned issues that I don't think have been looked at yet.

[Thaumcraft Crystal Creation with Akashic Tome/Morph-o-Tool in Arcane Workbench](https://github.com/focamacho/DupeFix-Project/issues/54)
[Magic Mirror causing NBT data loss](https://github.com/focamacho/DupeFix-Project/issues/11)

**Comment by TheCodex6824** [comment]
For the first issue, I looked at akashic tome and it has issues with the arcane workbench extra slots. It's not expecting the recipe to match while also having more than 1 other non-tome item in crafting inventory. I'm probably going to have to pass a fake crafting grid to all non-arcane recipes to resolve this.

For the second - yeah that's really not good. I also managed to have it get in an infinite loop and overflow the call stack, so that's a bit worrying.
